### PR TITLE
Add composer.json

### PR DESCRIPTION
**Pull request by ppp0**

### CODE REVIEW

**Comment by ppp0** [comment]
ha hoa

**Comment by tomaszdurka** [comment]
@njam was pizza good?

**Comment by tomaszdurka** [comment]
@njam im glad it was good!

File: build-ci.sh [hunk 1]
```diff
@@ -7,3 +7,7 @@ echo "✓"
 printenv
 for i in $(seq 1 10); do echo "$i"; sleep 1; done; echo
 echo "Done."
+echo
```
Comment (njam) [inline build-ci.sh]: @{7d7c0e0b-25a2-49ba-bfe9-94768ea36b50,Tomasz Durka} hello

**Pull request closed without merging**